omod-common/src/test/java/org/mockingbird/test/rest/resource/DuplicateNameAnimalResource_1_9.java

```diff
@@ -60,6 +61,21 @@ public DelegatingResourceDescription getRepresentationDescription(Representation
 		return null;
 	}
 	
+	@Override
+	public Model getGETModel(Representation representation) {
+		return null;
+	}
+	
```

Comment:
Yeah it's necessary that I write a documentation page, explaining how to document when introducing a new resource class. I will do that.